v2/snmp/serverhooks.go

```diff
@@ -32,12 +32,12 @@ var DefaultServerHooks = &ServerHooks{
 	},
 	WriteComplete: func(config *serverConfig, addr net.Addr, output []byte, err error) {
 		if err != nil {
-			log.Printf("WriteComplete target:%s err:%v\n", addr, err)
+			log.Printf("WriteDone target:%s err:%v\n", addr, err)
```

Comment:
should be WriteComplete?

Reply:
Reverted unintended changes to serverhooks.

Reply:
Reverted unintended changes to serverhooks.